src/server/editorServices.ts

```diff
@@ -583,7 +555,9 @@ namespace ts.server {
         }
 
         handleProjectFilelistChanges(project: Project) {
-            const { succeeded, projectOptions, error } = this.configFileToProjectOptions(project.projectFilename);
+            // TODO: Ignoring potentially returned 'error' and 'succeeded' condition

```

Comment:
I guess if there's a problem, it'll get fixed at that point. No sense leaving the comment.